src/data/markdown/docs/01 guides/02 Using k6/14 Executors.md

```diff
@@ -0,0 +1,295 @@
+
+| Option     | Type    | Description                                     | Default |
+|------------|---------|-------------------------------------------------|---------|
+| `vus`      | integer | Number of VUs to run concurrently.              | `1`     |
+| `duration` | string  | Total test duration (excluding `gracefulStop`). | -       |
+
+
+### Ramping VUs
+
+A variable number of VUs execute as many iterations as possible for a specified
+amount of time. This executor is equivalent to the global `stages` option.
+
+| Option             | Type    | Description                                                                   | Default |
+|--------------------|---------|-------------------------------------------------------------------------------|---------|
+| `startVUs`         | integer | Number of VUs to run at test start.                                           | `1`     |
+| `stages`           | array   | Array of objects that specify the target number of VUs to ramp up or down to. | `[]`    |
+| `gracefulRampDown` | string  | Time to wait for iterations to finish before starting new VUs.                | `"30s"` |
+
+
+### Externally controlled
+
+Control and scale execution at runtime via k6's REST API or the CLI.
+
+This executor formalizes what was [previously possible globally](https://k6.io/blog/how-to-control-a-live-k6-test),
+and using it is required in order to use the `pause`, `resume`, and `scale` CLI commands.
+
+| Option     | Type    | Description                                         | Default |
+|------------|---------|-----------------------------------------------------|---------|
+| `vus`      | integer | Number of VUs to run concurrently.                  | -       |
+| `duration` | string  | Total test duration.                                | -       |
+| `maxVUs`   | integer | Maximum number of VUs to allow during the test run. | -       |
+
+
+### Per VU iterations
+
+Each VU executes a fixed number of iterations.
+
+| Option        | Type    | Description                                                                    | Default |
+|---------------|---------|--------------------------------------------------------------------------------|---------|
+| `vus`         | integer | Number of VUs to run concurrently.                                             | `1`     |
+| `iterations`  | integer | Number of script iterations to execute with each VU.                           | `1`     |
+| `maxDuration` | string  | Maximum test duration before it's forcibly stopped (excluding `gracefulStop`). | `"10m"` |
+
+
+### Constant arrival rate
+
+A fixed number of iterations are executed in a specified period of time. This allows
+k6 to dynamically change the amount of VUs during a test run to achieve the specified
+amount of iterations per period, which is useful for a more accurate representation
+of RPS, for example. See [issue #550](https://github.com/loadimpact/k6/issues/550) for details.
```

Comment:
I reworded your suggestion a bit :) Let me know if f99fbf3 covers it.

Also, I guess the answer to my question [here](https://github.com/loadimpact/k6-docs/pull/49/files#diff-04ed84b1653e21012160c13d823f4a66R105) is that we should link to the arrival rate section you suggested in this same doc, instead of to #550.